another faulty integration test

`test_get_overrides_rss` will fail when the nvr contains characters not allowed in URLs.
 example:
```
AssertionError: assert [{'attrib': {}, 'tag': 'title', 'text': 'pythoncapi-compat-0^20221127git3779f12-2.el9'}, {'attrib': {}, 'tag': 'link', 'text': '[http://172.18.0.8:8080/overrides/pythoncapi-compat-0^20221127git3779f12-2.el9](http://172.18.0.8:8080/overrides/pythoncapi-compat-0%5E20221127git3779f12-2.el9)'}, {'attrib': {}, 'tag': 'description', 'text': 'Expedite unbundling in python-immutables'}, {'attrib': {}, 'tag': 'pubDate', 'text': 'Mon, 09 Jan 2023 19:30:50 +0000'}] in ....
```
because `pythoncapi-compat-0^20221127git3779f12-2.el9` link output is `overrides/pythoncapi-compat-0%5E20221127git3779f12-2.el9`

--- a/devel/ci/integration/tests/test_bodhi.py
+++ b/devel/ci/integration/tests/test_bodhi.py
@@ -886,7 +886,7 @@ def test_get_overrides_rss(bodhi_container, db_container):
             {
                 "tag": "link",
                 "attrib": {},
-                "text": f"http://{bodhi_ip}:8080/overrides/{override['nvr']}"
+                "text": f"http://{bodhi_ip}:8080/overrides/{quote(override['nvr'])}"
             },
             {"tag": "description", "attrib": {}, "text": override["notes"]},
             {"tag": "pubDate", "attrib": {}, "text": override["submission_date"]},

